Assets/Scripts/Spectator/SpectatorFollow.cs

```diff
@@ -0,0 +1,85 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Mirror;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+namespace PropHunt.Spectator
+{
+    /// <summary>
+    /// Script to track a spectatable object
+    /// </summary>
+    public class SpectatorFollow : NetworkBehaviour
+    {
+        /// <summary>
+        /// Action to move to next spectator object.
+        /// </summary>
+        [SerializeField]
+        [Tooltip("Minimum time between player follow target swaps")]
+        private UnityEngine.InputSystem.InputActionReference forwardAction;
+
+        /// <summary>
+        /// Action to move to previous spectator object.
+        /// </summary>
+        [SerializeField]
+        [Tooltip("Minimum time between player follow target swaps")]
+        private UnityEngine.InputSystem.InputActionReference backwardAction;
```

Comment:
we should change other things that use the input system to just use InputActionReference as it's much simpler to manage.